console/services/plugin/default_config.json

```diff
@@ -529,6 +529,24 @@
     "category": "exporter-plugin",
     "code_repo": "",
     "build_source": "image",
-    "image": "registry.cn-hangzhou.aliyuncs.com/goodrain/mysqld-exporter"
+    "image": "registry.cn-hangzhou.aliyuncs.com/goodrain/mysqld-exporter",
+    "config_group": [
+      {
+        "service_meta_type": "un_define",
+        "injection": "env",
+        "config_name": "配置",
+        "options": [
+          {
+            "attr_name": "DATA_SOURCE_NAME",
+            "attr_type": "string",
+            "attr_default_value": "${MYSQL_USERNAME}:${MYSQL_PASSWORD}@(127.0.0.1:${MYSQL_PORT:3306})/",
```

Comment:
MYSQL_USERNAME is usually an ordinary account. Is there any permission problem?

Reply:
If the permissions are not enough, the following commands need to be executed:
```
CREATE USER 'exporter'@'localhost' IDENTIFIED BY 'XXXXXXXX' WITH MAX_USER_CONNECTIONS 3;
GRANT PROCESS, REPLICATION CLIENT, SELECT ON *.* TO 'exporter'@'localhost';
```

Reply:
so? If the normal user does not work by default, should root be used by default.

Reply:
The environment variable of admin is MYSQL_USER. If MYSQL_USERNAME is not set, root will be used.

Reply:
ok